src/notation/notationmodule.cpp

```diff
@@ -172,6 +172,8 @@ void NotationModule::registerUiTypes()
 
 void NotationModule::onInit(const IApplication::RunMode&)
 {
+    Notation::init();
```

Comment:
Seems that this method should be called after s_configuration->init() because it uses configuration inside it